extensions_admin/setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='pulp_python_extensions_admin',
-    version='0.0.0',
+    version='1.0.0',
```

Comment:
This is slightly different than platform, which would be something like 1.0.0a1 I believe. I think both ways work OK though :koala: